readthedocs/subscriptions/constants.py

```diff
@@ -37,4 +38,5 @@
     (TYPE_AUDIT_LOGS, _("Audit logs")),
     (TYPE_AUDIT_PAGEVIEWS, _("Audit logs for every page view")),
     (TYPE_REDIRECTS_LIMIT, _("Redirects limit")),
+    (TYPE_FASTER_BUILDS, _("Faster builds")),
```

Comment:
Maybe we want a better name for this or a more granular description when listing this item in the UI about what it exactly it offers, "how much faster".

Reply:
Yeah, we already talk about the "faster builders" not being a good marketing name. We can use "Bigger builder instances" or similar.